p4-16/spec/P4-16-spec.mdk

```diff
@@ -4210,11 +4210,11 @@ Given a header stack value `hs` of size `n`, the following
 expressions are legal:
 
 - `hs[index]`: produces a reference to the header at the
-  specified position within the stack; if `hs` is an l-value, the
-  result is also an l-value. The header may be invalid. Some implementations
+  specified position within the stack; if `hs` is an l-value,
+  the result is also an l-value. The header may be invalid. Some implementations
```

Comment:
I would appreciate refraining from making just space changes, they add to the review burden significantly.